README.adoc

```diff
@@ -118,13 +116,13 @@ Auto-configuration will always back away as the user starts to define their own
 
 === spring-boot-starters
 Starters are a set of convenient dependency descriptors that you can include in your application.
-You get a one-stop-shop for all the Spring and related technology that you need without having to hunt through sample code and copy paste loads of dependency descriptors.
-For example, if you want to get started using Spring and JPA for database access include the `spring-boot-starter-data-jpa` dependency in your project, and you are good to go.
+You get a one-stop-shop for all the Spring and related technology you need without having to hunt through sample code and copy paste loads of dependency descriptors.
+For example, if you want to get started using Spring and JPA for database access, include the `spring-boot-starter-data-jpa` dependency in your project, and you are good to go.
 
 
 
 === spring-boot-cli
-The Spring command line application compiles and runs Groovy source, allowing you to write the absolute minimum of code to get an application running.
+The Spring command line application compiles and runs Groovy source, allowing you to write the absolute minimum code to get an application running.
```

Comment:
Make it be "minimum amount of code". Because code can have an amount, it should have "amount" in the sentence.